.github/workflows/live-tests.yml

```diff
@@ -13,36 +13,10 @@ on:
   workflow_dispatch: {}
 
 jobs:
-  main-branch:
+  berkeley:
     timeout-minutes: 45
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main' || (github.event_name == 'pull_request' && github.base_ref == 'main')
-    services:
-      mina-local-network:
-        image: o1labs/mina-local-network:o1js-main-latest-lightnet
-        env:
-          NETWORK_TYPE: 'single-node'
-          PROOF_LEVEL: 'none'
-        ports:
-          - 3085:3085
-          - 5432:5432
-          - 8080:8080
-          - 8181:8181
-          - 8282:8282
-        volumes:
-          - /tmp:/root/logs
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Use shared steps for live testing jobs
-        uses: ./.github/actions/live-tests-shared
-        with:
-          mina-branch-name: o1js-main
-
-  berkeley-branch:
-    timeout-minutes: 45
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/berkeley' || (github.event_name == 'pull_request' && github.base_ref == 'berkeley')
+    if: github.ref == 'refs/heads/main' || (github.event_name == 'pull_request')
```

Comment:
Why condition was changed by removing `&& github.base_ref == 'main'`?
If the goal is to always run the same job then no reason to have any conditions. But if you want to run it on PR against `main` branch then you need the complete condition.

Reply:
Ah ok . I guess the current conditions would run the job on `main`, and on PRs against `main`, `berkeley` and `develop`? That actually seems fine.
But I can also remove all conditions and run it on all 3 branches

Reply:
Without `&& github.base_ref == 'main'` in current state it wil run on any PR

Reply:
@shimkiv I don't think so because the intro has this:
```
  pull_request:
    branches:
      - main
      - berkeley
      - develop
```